charts/castai-db-optimizer/templates/service.yaml

```diff
@@ -17,8 +17,8 @@ spec:
     {{- range $index, $endpoint := .Values.endpoints}}
     {{- if empty $endpoint.name }}
     - name: endpoint-{{$index}}
-      port: {{$endpoint.port}}
-      targetPort: {{$endpoint.port}}
+      port: {{$endpoint.servicePort}}
+      targetPort: {{$endpoint.targetPort}}
```

Comment:
won't these need the reverted changed re-reverted first?

Reply:
Oh, you mean on the polyscale side? Yes.